addons/binding/org.openhab.binding.network/README.md

```diff
@@ -117,9 +117,13 @@ Linux has three different tools:
 *   arping of the ip-utils (Ubuntu/Debian: `apt-get install iputils-arping`)
 *   arping by Thomas Habets (Ubuntu/Debian: `apt-get install arping`)
 
-arping by Thomas Habets runs on Windows and MacOS as well.
+arping by Thomas Habets runs on Windows and MacOS as well. However versions older than 2.18 do not support an advantageous parameter combination (-C 1 -w <timeout>), thus making iputils-arping the preffered solution for raspbian/openHABian.
 
-Make sure the tool is available in the PATH, or in the same path as the openHAB executable.
+Make sure the tool is available in the PATH, or in the same path as the openHAB executable. When openHAB is running on Linux the environment variables, including PATH, can be displayed with the following command line:
+
+```
+cat /proc/$(ps -ax -o pid,user,command | grep -e '^[0-9]*[[:space:]]*openhab.*/usr/bin/java.*-Dopenhab.*$' | cut -f 1 -d ' ')/environ
```

Comment:
I can understand your point of view here: you want a complete self contained user guide on how to make openHAB work with this binding.

But please don't forget that there is an openHAB documentation as well. This paragraph about the Path doesn't belong here. You can reference to the related openHAB doc page or amend the doc if it doesn't contain such an information.